.github/workflows/process.yml

```
@@ -0,0 +1,57 @@
+name: Import Devices
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  import_devices:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+          cache: "pip"
+
+      - name: Install dependencies and extract CSV URL
+        id: csv_url
+        run: |
+          echo "Installing dependencies..."
+          pip install -r requirements.txt
+          echo "Extracting CSV URL from issue body..."
+          echo "csv_url=$(echo '${{ github.event.issue.body }}' | grep -oP 'http[s]?://[^)]+.csv')" >> $GITHUB_OUTPUT
+
+      - name: Download CSV file and import devices
+        id: import_devices
+        run: |
+          echo "Downloading CSV file..."
+          curl -L '${{ steps.csv_url.outputs.csv_url }}' -o to_process/data.csv
+          echo "Importing devices from CSV file..."
+          output=$(python script/process.py)
+          output="${output//$'\n'/'%0A'}"
+          echo "output=${output#*%0A%0A}" >> $GITHUB_OUTPUT
+
+      - name: Create Pull Request
+        id: create_pr
+        uses: peter-evans/create-pull-request@v6
+        with:
+          commit-message: "Import devices from issue #${{ github.event.issue.number }}"
+          title: "Import devices from issue #${{ github.event.issue.number }}"
+          body: "This pull request imports devices from a CSV file linked in issue #${{ github.event.issue.number }}.\n\n${{ steps.import_devices.outputs.output }}"
```

Add "Fixes" text, so it will auto-close old issue.
```suggestion
          body: "This pull request imports devices from a CSV file linked in issue #${{ github.event.issue.number }}. Fixes #${{ github.event.issue.number }}\n\n${{ steps.import_devices.outputs.output }}"
```